src/FontManager.h

```diff
@@ -21,6 +21,7 @@
  *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
  ***************************************************************************/
 
+#include <QMap>
```

Comment:
That Qt class `#include` will need AC's MSVC malloc-bug code workaround, i.e. the `#include "pre_guard.h"` and `#include "post_guard.h"` wrapping lines...

Reply:
Oh, is *that* what it does 😲!

Well since it is only applicable for those using the MSVC compiler I haven't had any interactions with it - I just know that it was placed around all other `#include <QtXxxxx>` instances and thought that that needed to be done here as well... so now it is sorted. 👍